fireant/slicer/queries.py

```diff
@@ -310,6 +343,18 @@ def _add_sorting(query, dimensions):
         return query
 
     @staticmethod
+    def _add_pagination(query, pagination, metrics, dimensions, references):
+        """ Add offset, limit and order pagination to the query """
+        query = query[pagination.offset: pagination.limit]
+        query_objects = utils.merge_dicts(metrics, dimensions, references)
+
+        for key, order in pagination.order:
+            field = query_objects.get(key)
+            query = query.orderby(field, order=order)
```

Comment:
Right now there is no way to do it. Maybe we should split these out into two different objects that can get passed in. The default behaviour as of master right now is to just sort the dimensions. What do you think?